src/components/Header/index.scss

```diff
@@ -0,0 +1,213 @@
+// Header top background-color #340EC1
+// Header search background-color #4D25E0
+
+
+
+.header_top{
```

Comment:
- i think it is better convention to use dash not underscore for word separation.